howso/utilities/installation_verification.py

```diff
@@ -433,7 +433,9 @@ def generate_dataframe(*, client: AbstractHowsoClient,
         default_action_features=action_features,
         persistence="never"
     )
-    trainee: Trainee = client.create_trainee(trainee_obj)
+    trainee = client.create_trainee(trainee_obj)
+    if not isinstance(trainee, Trainee):
+        raise Exception('Unable to create trainee.')
```

Comment:
More for personality curiosity, when do we raise HowsoErrors vs Exceptions?

Reply:
I'll change it.